src/NSwag.AspNetCore/Middlewares/RedirectToIndexMiddleware.cs

```diff
@@ -40,7 +40,7 @@ public async Task Invoke(HttpContext context)
 
                 var suffix = !string.IsNullOrWhiteSpace(_swaggerRoute) ? "?url=" + _transformToExternal(_swaggerRoute, context.Request) : "";
                 var path = _transformToExternal(_swaggerUiRoute, context.Request);
-                context.Response.Headers.Add("Location", (path != "/" ? path : "") + "/index.html" + suffix);
+                context.Response.Headers.Append("Location", (path != "/" ? path : "") + "/index.html" + suffix);
```

Comment:
Did you have a problem with the Add()?

Reply:
VS preview complains with https://learn.microsoft.com/en-us/aspnet/core/diagnostics/asp0019?view=aspnetcore-8.0